tool/resources/org/antlr/v4/tool/templates/codegen/Go/Go.stg

```diff
@@ -658,10 +655,10 @@ switch la_ {
 OptionalBlock(choice, alts, error) ::= <<
 p.SetState(<choice.stateNumber>)
 p.GetErrorHandler().Sync(p)
-la_ := p.GetInterpreter().AdaptivePredict(p.GetTokenStream(), <choice.decision>, p.GetParserRuleContext())
```

Comment:
❓ Have you tried changing this to use `=` instead of `:=`? It seems like you should able to forward declare this variable and then use it here without redeclaring it.